lib/common/actions.c

```diff
@@ -25,6 +25,62 @@
 #include <crm/common/xml.h>
 #include <crm/common/xml_internal.h>
 #include <crm/common/util.h>
+#include <crm/common/scheduler.h>
+
+/*!
+ * \brief Get string equivalent of an action type
+ *
+ * \param[in] action  Action type
+ *
+ * \return Static string describing \p action
+ */
+const char *
+pcmk_action_text(enum action_tasks action)
```

Comment:
Not sure this or `pcmk_on_fail_text()` needs to be public (or even that `pcmk_parse_action()` does but that one seems more likely). I have no strong feelings though. I could see them being potentially useful.

Reply:
I figure if the enum is public, the text/value mappings should be too. That also allows us to keep the string constants internal if nothing else needs them, since `..._text(value)` can get it